### PR TITLE
etcdctl: give more helpful suggestions on removal

### PR DESCRIPTION
**Pull request by barakmich**
/cc @xiang90 @robszumski @philips

### CODE REVIEW

File: etcdctl/command/member_commands.go [hunk 1]
```diff
@@ -156,16 +156,40 @@ func actionMemberRemove(c *cli.Context) {
...
+	}
+	// Sanity check the input.
+	foundID := false
+	for _, m := range members {
+		if m.ID == removalID {
+			foundID = true
+		}
+		if m.Name == removalID {
+			// Note that, so long as it's not ambiguous, we *could* do the right thing by name here.
+			fmt.Fprintf(os.Stderr, "Found a member named %s; if this is correct, please use its ID, eg:\n\tetcdctl member remove %s\n\n", m.Name, m.ID)
```
Comment (xiang90) [inline etcdctl/command/member_commands.go]: "eg:\n\tetcdctl member remove %s\n" this is dangerous. There might be duplicate name.

**Comment by robszumski** [comment]
LGTM, fixes https://github.com/coreos/etcd/issues/2195

File: etcdctl/command/member_commands.go [hunk 2]
```diff
@@ -156,16 +156,40 @@ func actionMemberRemove(c *cli.Context) {
...
+		}
+	}
+	if !foundID {
```
Comment (xiang90) [inline etcdctl/command/member_commands.go]: can we move this to line 165? if the error is 404, print out the error

File: etcdctl/command/member_commands.go (hunk 1 above)
Comment (barakmich) [inline etcdctl/command/member_commands.go]: If there actually is, you'll get two of these messages.

File: etcdctl/command/member_commands.go (hunk 2 above)
Comment (barakmich) [inline etcdctl/command/member_commands.go]: I think you mean line 190. But that error is useless!

File: etcdctl/command/member_commands.go (hunk 1 above)
Comment (yichengq) [inline etcdctl/command/member_commands.go]: @barakmich Can you refer it to the doc too? https://github.com/coreos/etcd/blob/master/Documentation/runtime-configuration.md#remove-a-member
It is more helpful if they know what the exact workflow is.

Comment (xiang90) [inline etcdctl/command/member_commands.go]: @yichengq Agreed. I am **REALLY** afraid people would do this operation without looking through the doc.

**Pull request merged**